devtools/buildVisualStories.js

```diff
@@ -44,12 +44,10 @@ export default {
 export const ${componentName} = () => {
```

Comment:
line 27 says
>             // TODO: reenable storyshots for examples using hooks in storybook@6
            // https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.43

Can we do that now?

Reply:
Yes!

Reply:
Dialog and Calendar are still having problems so I've left them in the ignored bucket for now and will do a separate followup